bin/node-gyp.js

```diff
@@ -25,14 +25,18 @@ prog.parseArgv(process.argv)
 prog.devDir = prog.opts.devdir
 
 var homeDir = osenv.home()
-if (prog.devDir) {
+if (process.env.LOCALAPPDATA) {
+  // TODO: create intermediate directories
+  pro.devDir = path.resolve(process.env.LOCALAPPDATA, 'nodejs', 'node-gyp')
```

Comment:
Does `path.resolve` add the directory separators for me?  The documentation is a bit vague on this.